### DISCUSSION TITLE
rmview config environment variable

### DISCUSSION THREAD

**Discussion by RDeRenzi** [bordaigorl/rmview#59] [Q&A]
Sorry if this is trivial.
Is there a bash environment variable that I can use for the json configuration file?
if I place rmview and its json file in my local bib directory it works
if I create a rmview.desktop in e.g. /usr/share/applications/ how can I direct it to the proper config json file?

Thanks a lot to anybody who knows
Roberto
PS rmview saves my life, thanks!!

**Comment by bordaigorl** [comment]
You can pass the path of the config file as an argument, or you can set the environment variable `RMVIEW_CONF`.
